src/util.cpp

```diff
@@ -141,7 +143,7 @@ void Util::parseArgs(int argc,
 #if USE_MMAL_ENCODER || USE_JETSON_ENCODER
         return std::string();
 #else
-        return "このデバイスは --force-i420 に対応していません。";
+        return "This machine does NOT support --force-i420 option.";
```

Comment:
cannot use --force-i420 option because your device does not have this feature.

Reply:
device むずかしいですね。そもそもランタイム capability じゃなくてビルドフラグなのですよね。どうしようかな。

Reply:
これはまるっと採用します :+1:

Reply:
メッセージの表示され方を見るとちょっと冗長だったので変えてみました。
↓のようにしてみています。

```
(default) % ./momo --force-i420                                                                                                                                                                                      --force-i420: Not available because your device does not have this feature.                                                                                                                                          Run with --help or --help-all for more information.    
```